bin/collator/res/shiden.raw.json

```diff
@@ -1,8 +1,11 @@
 {
-  "name": "Shiden Shell",
+  "name": "Shiden",
```

Comment:
Just a heads up, the chain page metadata in Polkadot-js apps portal still points to `Shiden Shell`, please update them as well when this network goes live.

- https://github.com/polkadot-js/apps/blob/f5a2991d2ac1e56f48bb0d04718fb2a255969a25/packages/apps-config/src/links/subscan.ts#L27
- https://github.com/polkadot-js/apps/blob/fe3a4f18947620b14ba958d03a775f38c00eb610/packages/apps-config/src/ui/colors.ts#L259
- https://github.com/polkadot-js/apps/blob/fe3a4f18947620b14ba958d03a775f38c00eb610/packages/apps-config/src/ui/logos/index.ts#L216